src/test/java/seedu/storemando/logic/parser/ReminderCommandParserTest.java

```diff
@@ -25,11 +25,11 @@ public void parse_invalidSingleArg_throwsParseException() {
             String.format(MESSAGE_INVALID_COMMAND_FORMAT, ReminderCommand.MESSAGE_USAGE));
     }
 
-    @Test
-    public void parse_numberSmallerThanZero_throwsParseException() {
-        assertParseFailure(parser, "-1 day",
-            String.format(MESSAGE_INVALID_COMMAND_FORMAT, ReminderCommand.MESSAGE_USAGE));
-    }
+    //    @Test
+    //    public void parse_numberSmallerThanZero_throwsParseException() {
```

Comment:
Can just delete the test. Don't need to comment it out

Reply:
I added the test back.